### PR TITLE
Fix ratio rendering

### PR DESCRIPTION
**Pull request by dknochen**
fixing issue #123

### CODE REVIEW

**Comment by dknochen** [comment]
@wcandillon ready for review. 

I had to increase the timeout again. It wouldn't work otherwise. Maybe, you have an idea to make it faster (This filter implementation is the culprit: https://github.com/28msec/nolap-report-editor/pull/137/files#diff-6459ecefe1b9c78093daa5a4428d36d5R16 ).

File: app/report/spreadsheet/spreadsheet-page.js [hunk 1]
```diff
@@ -4,14 +4,38 @@
...
 }
 
 Spreadsheet.prototype.visitPage = function(){
     return browser.get('/' + this.id + '/spreadsheet');  
 };
 
+Spreadsheet.prototype.getValuesByHeaderContainingText = function(containedText){
+    return this.dataRows
+        .filter(function(elem){
+            return elem.element(by.css('td.header > span'))
```
Comment (wcandillon) [inline app/report/spreadsheet/spreadsheet-page.js]: Maybe cssContainingText() can be used there or xpath().

File: tests/e2e/report-scenario.js [hunk 2]
```diff
@@ -36,4 +36,31 @@ describe('Report', function(){
         expect(recompilePage.errorMessages.count()).toBe(0);
         expect(recompilePage.successMessages.count()).toBe(66);
     });
+
+    it('should select Coca Cola on filters page', function(){
+        report.filters.visitPage()
```
Comment (wcandillon) [inline tests/e2e/report-scenario.js]: Let's try to use then() only when necessary.

**Comment by wcandillon** [comment]
Beautiful!

**Pull request merged**